src/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.sarif

```diff
@@ -2457,6 +2457,25 @@
             ]
           }
         },
+        "CA1835": {
+          "id": "CA1835",
+          "shortDescription": "Prefer 'ReadOnlyMemory<Byte>' overload for 'WriteAsync'.",
```

Comment:
@carlossanlop I think you may want to update the Title and Description for CA1835 rules to be shared, i.e. something like `Prefer 'ReadOnlyMemory<Byte>' overload for 'ReadAsync'/'WriteAsync'` and use it for both your descriptors.